Makefile

```diff
@@ -1,14 +1,12 @@
 BUILD_CHANNEL?=local
 TOOL_BIN = bin/gotools/$(shell uname -s)-$(shell uname -m)
 PATH_WITH_TOOLS="`pwd`/$(TOOL_BIN):${PATH}"
-
-set-pkg-config-openssl:
-	pkg-config openssl || export PKG_CONFIG_PATH=$$PKG_CONFIG_PATH:`find \`which brew > /dev/null && brew --prefix\` -name openssl.pc | head -n1 | xargs dirname`
```

Comment:
This didn't actually work; the syntax for setting the PKG_CONFIG_PATH was wrong and exporting an environment variable in a Makefile like this will not persist the value since in a Makefile each line is executed in a different subshell, per [this thread](https://unix.stackexchange.com/questions/656437/export-env-variable-does-not-work-from-makefile) on Stack Exchange.